api/src/graphql/users.sdl.ts

```diff
@@ -26,9 +26,10 @@ export const schema = gql`
 
   input CreateUserInput {
     email: String!
-    name: String
-    agencyId: Int
-    role: RoleEnum
+    name: String!
+    agencyId: Int!
+    role: RoleEnum!
+    isActive: Boolean
```

Comment:
According to the ticket's requirement `isActive` field on `CreateUserInput` should be required. However, this means that we either need to pass `isActive: true` from the frontend (currently the form for creating users doesn't have this field), or make this field optional on `CreateUserInput`, allowing `schema.prisma` to set it to true by default.
If we would rather set it on the frontend, I can make this change.